pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -29,15 +29,15 @@ import (
 	"github.com/awslabs/karpenter/pkg/cloudprovider/aws/utils"
 	"github.com/awslabs/karpenter/pkg/utils/parallel"
 	"github.com/awslabs/karpenter/pkg/utils/project"
-	"go.uber.org/zap"
 	v1 "k8s.io/api/core/v1"
 	"knative.dev/pkg/apis"
+	"knative.dev/pkg/logging"
 )
 
 const (
 	// CreationQPS limits the number of requests per second to CreateFleet
 	// https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits
-	CreationQPS = 2
+	CreationQPS = 10
```

Comment:
why are we raising this to 10?  I think raising burst makes sense, but I don't think raising this to 10 is a good idea, maybe 5 ?